alacritty/build.rs

```diff
@@ -22,6 +22,10 @@ use std::path::Path;
 use embed_resource;
 
 fn main() {
+    if cfg!(not(any(feature = "x11", feature = "wayland", target_os = "macos", windows))) {
+        panic!("Please select at least one Linux/BSD backend!");
```

Comment:
Minor nitpick, but I think I'd word this to include the word "feature" to make it a bit more clear to the user. Perhaps just adding the word feature to the end of the sentence.

Also, an exclamation seems a bit dramatic, don't you think? Most of out messages don't have punctuation, we should leave it off here too.